airflow/providers/databricks/example_dags/example_databricks.py

```diff
@@ -71,4 +72,17 @@
         libraries=[{'jar': 'dbfs:/lib/etl-0.1.jar'}],
     )
     # [END howto_operator_databricks_named]
-    notebook_task >> spark_jar_task
+
+    # [START howto_operator_databricks_pyspark]
```

Comment:
These `# [START/END howto_operator_...]` tags are used as references in the operator documentation so the docs can display code snippets of the related example DAG between these `START` and `END` tags. Using these tags keeps the examples and the operator docs in sync. Would you be willing to add some context around this new task and how it relates to the `DatabricksSubmitRunOperator `?

You can check out how similar tags are used for the existing operators in the Databricks operator documentation [here](https://github.com/apache/airflow/edit/main/docs/apache-airflow-providers-databricks/operators.rst).

Reply:
Oh yeah That should be indeed updated in the Databricks howto: https://airflow.apache.org/docs/apache-airflow-providers-databricks/stable/operators.html

Reply:
@josh-fell @potiuk Thanks for the pointer... I'll see about making that update.

Reply:
My colleague has an open [PR](https://github.com/apache/airflow/pull/19156) that would overlap with this one. I'd like to withdraw this PR and re-submit once the other PR is merged.